publish.gradle

```diff
@@ -35,30 +35,27 @@ nexusPublishing {
 	}
 }
 
-task javadocJar(type: Jar) {
-	classifier = 'javadoc'
-	from javadoc
-}
-
-task sourcesJar(type: Jar) {
-	classifier = 'sources'
-	from sourceSets.main.allSource
-}
-
-artifacts {
-	archives javadocJar, sourcesJar
+// to remove shadowed jar from the regular release publication
+components.java.withVariantsFromConfiguration(configurations.shadowRuntimeElements) {
+	skip()
 }
 
 afterEvaluate {
 	publishing {
+		repositories {
+			maven {
+				url 'https://repo.repsy.io/mvn/ali200170/default'
+				credentials {
+					username 'ali200170'
+					password 'qcLNJweyo3v3f8'
+				}
+			}
+		}
```

Comment:
What do you think about hiding those credential keys using the GitHub Actions?

Reply:
oh, very nice catch. fixing it.

Reply:
Fixed this.